app/mi-bot.py

```diff
@@ -97,7 +98,9 @@ def episode_titles(self):
         return [i.title for i in self.feed['items']]
 
 
-mi_feed = PodcastFeed(url=MINKORREKT_RSS, dump=DUMP)
+podcast_feed = PodcastFeed(url=PODCAST_FEED, dump=DUMP)
```

Comment:
Might be better to keep the minkorrekt feed as a fallback if no env var is set

Reply:
If the env var is not set, line 27 will raise a `KeyError` and the bot won't start.

This is the same as if not TOKEN would be given.

Or do you just want to keep backwards compatibility for the config?